string[] tasks
float64[] error